src/ngraph/op/interpolate.cpp

```diff
@@ -32,8 +31,21 @@ op::Interpolate::Interpolate(const Output<Node>& image,
     constructor_validate_and_infer_types();
 }
 
+bool op::Interpolate::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("attrs.axes", m_attrs.axes);
+    visitor.on_attribute("attrs.mode", m_attrs.mode);
+    visitor.on_attribute("attrs.align_corners", m_attrs.align_corners);
+    visitor.on_attribute("attrs.antialias", m_attrs.antialias);
+    visitor.on_attribute("attrs.pads_begin", m_attrs.pads_begin);
+    visitor.on_attribute("attrs.pads_end", m_attrs.pads_end);
+    return true;
+}
+
 void op::Interpolate::validate_and_infer_types()
 {
+    // Check if string attribute is valid by converting to and back from valid enum type
```

Comment:
Take out this check. Something will be passing the string incorrectly and count on it not getting an error and this will be considered breaking API.

Reply:
Done.